docs/components/develop.md

```diff
@@ -23,31 +26,60 @@ Start with the ODTP component template:
 
 - go to [`odtp-component-template`](https://github.com/odtp-org/odtp-component-template)
 - Click on "Use this template": "Open a new repository"
-- Give the component a name similar to "odtp-name of your tool"
+- Give the component a name similar to "odtp-your-tool-name"
 
 !!! Note
     This repository makes use of submodules. Therefore, when cloning it you need to include them.
     
     ```bash 
-    git clone --recurse-submodules https://github.com/odtp-org/odtp-component-template
+    git clone --recurse-submodules https://github.com/odtp-org/odtp-your-tool-name
```

Comment:
@sabinem, The name of the organization can be changed here, too. People will fork in their profiles or organizations.